### ISSUE TITLE
```devbox services stop``` stops php services of all running projects

### ISSUE THREAD

**Issue by Vandaahl** [jetpack-io/devbox#783]
I have two similar projects with this setup:

```
{
  "packages": [
    "php81",
    "php81Packages.composer",
    "apacheHttpd",
    "nodejs-16_x",
    "php81Extensions.apcu"
  ],
  "env": {
    "HTTPD_PORT": "8000",
    "HTTPD_SECURE_PORT": "4000",
    "PHPFPM_PORT": "9000",
    "DOCUMENT_ROOT": "${PWD}/public"
  },
  "shell": {
    "init_hook": []
  },
  "nixpkgs": {
    "commit": "ee01de29d2f58d56b1be4ae24c24bd91c5380cea"
  }
}
```
Each have their own php-fpm service and they can run simultaneously. But when I run ```devbox services stop``` for one project, php-fpm is stopped for both. Apache seems unaffected. Is this a configuration error on my part?

**Comment by gcurtis** [comment]
@mikeland86 you probably have the most context on this.

**Comment by Lagoja** [comment]
This should be addressed with the new services functionality in 0.4.7 -- `devbox stop` should only affect the running processes/services for a single project
